src/ai/ai_gui.cpp

```diff
@@ -226,25 +236,12 @@ struct AIConfigWindow : public Window {
 				break;
 			}
 
-			case WID_AIC_DECREASE_INTERVAL:
```

Comment:
I don't think we want to remove the `WID_AIC_DECREASE_INTERVAL` and `WID_AIC_INCREASE_INTERVAL` cases, since these handle the functionality of the "Interval between starting of competitors:" buttons.